code/modules/antagonists/blob/blobstrains/synchronous_mesh.dm

```diff
@@ -15,10 +15,10 @@
 	if(damage_flag == MELEE || damage_flag == BULLET || damage_flag == LASER) //the cause isn't fire or bombs, so split the damage
 		var/damagesplit = 1 //maximum split is 9, reducing the damage each blob takes to 11% but doing that damage to 9 blobs
 		for(var/obj/structure/blob/C in orange(1, B))
-			if(!istype(C, /obj/structure/blob/core) && !istype(C, /obj/structure/blob/node) && C.overmind && C.overmind.blobstrain.type == B.overmind.blobstrain.type) //if it doesn't have the same chemical or is a core or node, don't split damage to it
+			if(!istype(C, /obj/structure/blob/special/core) && !istype(C, /obj/structure/blob/special/node) && C.overmind && C.overmind.blobstrain.type == B.overmind.blobstrain.type) //if it doesn't have the same chemical or is a core or node, don't split damage to it
```

Comment:
If you could substitute these type checks for properties stored in variables, that'd be a code improvement.

Reply:
Not entirely sure what you mean here, store what property in variables?